### ISSUE TITLE
0crat pay +15 for PR

### ISSUE THREAD

**Issue by DronMDF** [zerocracy/datum#94]
https://github.com/DronMDF/telebot/pull/5

**Comment by 0crat** [comment]
@yegor256 please, pay attention to this issue

**Comment by 0crat** [comment]
Job `gh:zerocracy/datum#94` assigned to @g4s8, please go ahead ([policy](http://datum.zerocracy.com/pages/policy.html)).

**Comment by DronMDF** [comment]
This is not a problem in this time. Already fixed?
 I close this issue?

**Comment by g4s8** [comment]
@DronMDF no, this bug is not fixed yet. Same for #96. I'm going to fix it after jcabi/jcabi-github#1324 that is blocked by jcabi/jcabi-github#1325

**Comment by DronMDF** [comment]
Ok, wait for fix.

**Comment by g4s8** [comment]
@DronMDF it should be fixed in zerocracy/farm#155, you can check it.

**Comment by DronMDF** [comment]
@g4s8 Thank's. I'll try it in weеk.

**Comment by 0crat** [comment]
@g4s8 resigned from `gh:zerocracy/datum#94`, please stop working.

**Comment by 0crat** [comment]
Job `gh:zerocracy/datum#94` assigned to @caarlos0, please go ahead ([policy](http://datum.zerocracy.com/pages/policy.html)).

**Comment by 0crat** [comment]
Oops! You need to have one of these roles in order to do what you're trying to do: ARC, PO. I'm sorry to say this, but at the moment you've got no roles in this project (your GitHub login is "0crat").

**Comment by 0crat** [comment]
@caarlos0 resigned from `gh:zerocracy/datum#94`, please stop working.

**Comment by yegor256** [comment]
@DronMDF please, check, it's fixed

**Comment by DronMDF** [comment]
@yegor256 This is ok. Thank's.

**Comment by 0crat** [comment]
Oops! There is no order for job `gh:zerocracy/datum#94`.

**Comment by 0crat** [comment]
Job `gh:zerocracy/datum#94` is now out of scope.
